packages/core/src/Stores/client-store.js

```diff
@@ -613,10 +618,21 @@ export default class ClientStore extends BaseStore {
         return this.dxtrade_accounts_list.some(account => account.account_type === 'real');
     }
 
+    get has_real_derivez_login() {
+        return this.derivez_accounts_list.some(account => account.account_type === 'real');
+    }
+
     hasAccountErrorInCFDList = (platform, account_type) => {
         if (!this.is_logged_in) return false;
 
-        const list = platform === CFD_PLATFORMS.MT5 ? this.mt5_login_list : this.dxtrade_accounts_list;
+        const list =
+            platform === CFD_PLATFORMS.MT5
+                ? this.mt5_login_list
+                : platform === CFD_PLATFORMS.DXTRADE
+                ? this.dxtrade_accounts_list
+                : platform === CFD_PLATFORMS.DERIVEZ
+                ? this.derivez_accounts_list
+                : '';
```

Comment:
```suggestion
        let list = '';

        switch (platform) {
          case CFD_PLATFORMS.MT5:
             list = this.mt5_login_list;
             break;
         case CFD_PLATFORMS.DXTRADE:
           list = this.dxtrade_accounts_list;
           break;
        case CFD_PLATFORMS.DERIVEZ:
          list = this.derivez_accounts_list;
          break;
        default:
           list = '';
          break;
 }
```
avoid multiple ternaries and try to use if else or switch case

Reply:
I have already created a function to fix this issue.
Will send its link to you @shontzu-deriv